### PR TITLE
High DPI landing images for PR 9879

### PR DESCRIPTION
**Pull request by bene-dictator**
~~Apparently "can't automatically merge". Github buggin?~~

Outdated master lmao

### CODE REVIEW

**Review by ziproot — COMMENTED** [review]
I ran this through debian-control-linter. There are some errors unrelated to this PR, but some are still issues
image

**Comment by bene-dictator** [comment]
Maybe solved?

File: copyright [hunk 1]
```diff
@@ -1998,6 +2008,36 @@ Copyright: Daeridanii (https://github.com/Daeridanii1)
 License: CC0
 Comment: selected and unselected versions of images/ui/find, by Ember369 (https://github.com/Ember369)
 
+Files:
+ images/land/haze0*
+Copyright: Izaac Hewitt-Lawrence
+License: Unsplash
+ Taken from unsplash.com and cropped. This image was
```
Comment (ziproot) [inline copyright:2015]:
```suggestion
 Taken from https://archive.is/LZQbZ and cropped. This image was
```

File: copyright [hunk 2]
```diff
@@ -1998,6 +2008,36 @@ Copyright: Daeridanii (https://github.com/Daeridanii1)
...
+ uploaded to unsplash.com after June 2017, so it is subject
+ to the Unsplash License. It was also uploaded after February
+ 2018, so it is subject to an additional restriction limiting
+ the sale of unaltered copies.
+
+Files:
+ images/land/haze1*
+Copyright: Tobias Rademacher
+License: Unsplash
+ Taken from unsplash.com and cropped. This image was
```
Comment (ziproot) [inline copyright:2025]:
```suggestion
 Taken from https://archive.is/Rc9fs and cropped. This image was
```

File: copyright [hunk 3]
```diff
@@ -1998,6 +2008,36 @@ Copyright: Daeridanii (https://github.com/Daeridanii1)
...
+ uploaded to unsplash.com after June 2017, so it is subject
+ to the Unsplash License. It was also uploaded after February
+ 2018, so it is subject to an additional restriction limiting
+ the sale of unaltered copies.
+
+Files:
+ images/land/haze2*
+Copyright: Marek Piwnicki
+License: Unsplash
+ Taken from unsplash.com and cropped. This image was
```
Comment (ziproot) [inline copyright:2035]:
```suggestion
 Taken from https://archive.is/iVNGc and cropped. This image was
```

**Review by ziproot — COMMENTED** [review]
snow23 is a duplicate of scene/iceplains3

File: copyright [hunk 4]
```diff
@@ -3018,6 +3018,17 @@ Comment:
  so it is subject to an additional restriction limiting the
  sale of unaltered copies.
 
+Files:
+ images/land/snow23*
+Copyright: Daniel Frank
+License: Unsplash
+Comment:
+ Taken from unsplash.com and cropped. This image was
```
Comment (ziproot) [inline copyright:3026]:
```suggestion
 Taken from https://archive.is/xK5Wu and cropped. This image was
```

**Review by ziproot — APPROVED** [review]

**Pull request merged**